kolibri/content/api.py

```diff
@@ -24,6 +26,28 @@ class ChannelMetadataCacheViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
         return models.ChannelMetadataCache.objects.all()
 
+    def destroy(self, request, pk=None):
```

Comment:
How are permissions handled on here? Can *any* user do this? Or is there an implicit "superuser only" permission?